cmd/config_test.go

```diff
@@ -98,3 +100,128 @@ func TestTLSConfigLoad(t *testing.T) {
 		})
 	}
 }
+
+func TestTemporalSetup(t *testing.T) {
```

Comment:
If you add the `len(ts.Shards) == 0` error check I suggested earlier in my review this test should get a new testcase like:
```
ts: TemporalSet{
  Name: "whatever",
},
err: "TemporalSet can't be empty",
```

Reply:
Done.